xsdata/parser.py

```diff
@@ -115,6 +116,50 @@ def set_schema_namespaces(self, obj: xsd.Schema, element: etree.Element):
             if namespace.uri not in namespaces:
                 obj.nsmap[namespace.prefix] = namespace.uri
 
+    def add_default_imports(self, obj: xsd.Schema):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5137871)